main.py

```diff
@@ -0,0 +1,24 @@
+"""
+Program Name: Credit Union Accounts
+Author: Rebecca Davidson
+Date: 3/20/2023
+Description: This program will display a member's bank account summary including balances and subshares
+"""
+
+
+class Account:
+    """
+    Credit Union Account
+    """
+
+    def __init__(self, account_number, full_name):
+        self.__account_number = account_number
+        self.full_name = full_name
+
+    def account_summary(self):
+        return f"{self.full_name}'s account number is {self.__account_number}"
+
+
+if __name__ == "__main__":
+    member = Account("123456", "Luis Santiago")
+    print(member.account_summary())
```

Comment:
## Week 8 Grading

| Points | Location        | Criteria                                    |
| ------ | --------------- | ------------------------------------------- |
| 10     | main.py         | Protected variable set when class initiated |
| 10     | main.py         | Function that uses protected variable       |
| 10     | pytest coverage | Tests cover all aspects of main.py          |
| 8     | pylint          | Pylint score >= 8.0.                        |
|        | instructor      | Code style, efficiency, and readability     |
| 38/40 |

For now, focus on these [pylint errors](https://github.com/immaculata-cis-218/python-classes-rdavidsonimmaculata/actions/runs/4522049947/jobs/7964223410):
```
main.py:5:0: C0301: Line too long (103/100) (line-too-long)
main.py:18:4: C0116: Missing function or method docstring (missing-function-docstring)
```
`black .` might help you out here